packages/web/src/styles/Input.js

```diff
@@ -146,6 +146,13 @@ const suggestions = (themePreset, theme) => css`
 		&:focus {
 			background-color: #eee;
 		}
+
```

Comment:
@ShahAnuj2610 You defined the class under `suggestions`, I don't think it'll work for `ReactiveList`.
You can create a styled component & use it.

Reply:
yeah, right! But I can't use styled component since I am using `dangerouslysetInnerHtml`, so will be using emotion's `injectGlobal`. Thoughts @bietkul @jyash97?